### ISSUE TITLE
AVT 3 - React FileUploader X icon is not announced by JAWS and it does not clear the selection

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3437]
## Environment
Windows 10
FireFox Quantum 68.0
JAWS 18

## Description
1. Tab to the Add Files button. JAWS announces "Main region, add files, clickable."
2. Press Space or Enter to open the dialog, navigate to a jpg file and select it.
3. Tab to the X icon, JAWS announces "Provide item description."
**Expected Result:** JAWS should announce the purpose of the button, such as "Clear File"
4. Press Enter or Space on the icon and the file fails to clear the selected file. 
Note: The icon works as expected for keyboard-only users.
